docs/env.md

```diff
@@ -64,11 +64,15 @@ If not set, then default (`console.info` and above) logging is enabled.
 If set to an empty string, or running in `ag-chain-cosmos start` mode, don't
```

Comment:
Please also take care to preserve the semantics in the sentence above this one:

> If not set, then default (`console.info` and above) logging is enabled.
(`console.log` and `console.debug` logging is disabled.)

So, `process.env.DEBUG === undefined` is treated differently than `process.env.DEBUG === ''`.

Setting `DEBUG=""` is supposed to silence all logging, but not setting `DEBUG` at all should give default logs.

Reply:
I think I have. PTAL.

However, this convention has a problem. Adding or removing unrelated members of the `DEBUG` list, like `'label-instances'`, will effect this log silencing, which is surprising. What should we do about this surprise?

Reply:
> However, this convention has a problem. Adding or removing unrelated members of the `DEBUG` list, like `'label-instances'`, will effect this log silencing, which is surprising.

Ohhh, not good.

> What should we do about this surprise?

I'd propose changing the convention to be `DEBUG=agoric:${level}`.  So, we'd use `DEBUG=agoric:none` instead of `DEBUG=""` to indicate silencing.  `DEBUG=agoric` would be the backward-compatible verbose setting, as would `DEBUG=agoric:debug`.  An unset `$DEBUG` would be equivalent to the default `DEBUG=agoric:info`.

The files that need to be modified are the `env.md` itself, and `anylogger-agoric.js` (there's two copies of this in agoric-sdk).  The other places that need updating can be found by searching for `DEBUG` in `packages/agoric-cli` and `packages/solo`.

Reply:
You did these, so resolving.